core/lib/common/worker.js

```diff
@@ -200,12 +234,19 @@ module.exports = class Worker {
 		return rp.get({ uri: `${this.url}/contract`, json: true });
 	}
 
-	capture(action) {
+	async capture(action) {
 		switch (action) {
 			case 'start':
 				return rp.post({ uri: `${this.url}/dut/capture`, json: true });
 			case 'stop':
-				return request.get({ uri: `${this.url}/dut/capture` });
+				// have to receive tar.gz and unpack them? then return path to directory for the test to consume
+				let capture = request.get({ uri: `${this.url}/dut/capture` });
+				const line = pipeline(
```

Comment:
Nitpick:
- No point having a catch handler that rethrows the error here, especially when the promise is awaited immediately after. Just return the promise. Not a blocking suggestion, just a style/readability thing.

As an example, instead of doing:
```JavaScript
async foo() {
  const bar = baz().catch(e => {throw e});
  await bar;
}
```

You can do instead:
```JavaScript
async foo() {
    return baz();
}